src/fsharp/FSharp.Core/array.fs

```diff
@@ -120,8 +121,8 @@ namespace Microsoft.FSharp.Collections
             
         [<CompiledName("Replicate")>]
         let replicate count x = 
-            if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
-            let arr = (Microsoft.FSharp.Primitives.Basics.Array.zeroCreateUnchecked count : 'T array)  
+            if count < 0 then invalidArg "count" LanguagePrimitives.ErrorStrings.InputMustBeNonNegativeString
```

Comment:
I think we don't want to do this for non-inlined functions.  There is an outstanding bug where those "inlined" strings are non-localized (e.g. in Italian).  So where possible we use the SR.GetString call